prime-router/src/main/kotlin/Report.kt

```diff
@@ -28,7 +28,7 @@ class Report {
 
     val rowCount: Int get() = this.table.rowCount()
     val rowIndices: IntRange get() = 0 until this.table.rowCount()
-    val name: String get() = formFileName(id, schema.baseName, createdDateTime)
+    val name: String get() = formFileName(id, schema.baseName, destination?.format, createdDateTime)
```

Comment:
There were two separate places where blob filenames were being created.   I've consolidated them into one static function in Report, formFileName.  (This is why the .csv extension wasn't working)